test/functional/rpc_fundrawtransaction.py

```diff
@@ -1143,6 +1144,21 @@ def test_feerate_rounding(self):
         rawtx = w.createrawtransaction(inputs=[], outputs=[{self.nodes[0].getnewaddress(address_type="bech32"): 1 - 0.00000202}])
         assert_raises_rpc_error(-4, "Insufficient funds", w.fundrawtransaction, rawtx, {"fee_rate": 1.85})
 
+    def test_sffo_with_negative_ev(self):
+        self.log.info("Testing that subtract fee from outputs with negative effective value outputs still works")
+
+        self.nodes[1].createwallet("negevsffotest")
+        w = self.nodes[1].get_wallet_rpc("negevsffotest")
```

Comment:
assert `w` has no balance/utxos, may seem duh but it helps readability of the two wallet dance

Reply:
Done